### DISCUSSION TITLE
Sheet Rating

### DISCUSSION THREAD

**Discussion by LucasAlbuquerque0** [Telefonica/mistica-design#1567] [Proposals]
### Goal

Have a sheet component for ranking experiences.

### Context

This component is necessary in several areas of the application, ensuring ranking for purchases, services, subscriptions, etc. The main request was for subscriptions and services, capturing the results of the user experience more quickly.

### Impact

Facilitate receiving feedback at different levels, eliminating the need for an entire screen to capture data.

### Discussion Questions

Can we use Rating at more than one level of information? Example Rank with stars and capture information for low grades such as "Why did you give this grade?"

### Additional considerations

There is a proposal for a Custom Sheet (Slot) and a Rating component, using these components it is possible to achieve the same result.

[Link para Jira da proposta](https://jira.tid.es/browse/OBVIVO-2123)

### Do you have a visual proposal?

https://www.figma.com/file/3FWICps6yc9do1FVBRxxgN/Bottom-Sheet-Evolu%C3%A7%C3%A3o?type=design&node-id=331%3A5822&mode=design&t=O88Dl97T9UnDeXan-1

**Comment by yceballost** [comment]
I think the dev team is looking for a solution to have webviews in fullscreen. if this possibility is carried out, we will can use custom sheets using web sheets inside a hybrid application like Meu Vivo (Novum)

we are waiting to hear what happens with this from Novum side

---

From the other side, we need to discuss about Rating component, I think is better in another discussion. But we are agree to include this component in Mística

**Reply by LucasAlbuquerque0** [reply]
Perfect! When discussions about the Rating will take place. Could you include me?

**Reply by yceballost** [reply]
of course! you can open this discussion if you want and include different possibilities of this component

**Comment by yceballost** [comment]
Issue created for rating component #1623

The problem with the Sheet is something to work in the Novum side, not from Mística. I will close this discussion.
